x-pack/plugins/enterprise_search/common/constants.ts

```diff
@@ -188,8 +188,6 @@ export const APP_SEARCH_URL = '/app/enterprise_search/app_search';
 export const ENTERPRISE_SEARCH_ELASTICSEARCH_URL = '/app/enterprise_search/elasticsearch';
 export const WORKPLACE_SEARCH_URL = '/app/enterprise_search/workplace_search';
 
-export const ENTERPRISE_SEARCH_DOCUMENTS_DEFAULT_DOC_COUNT = 25;
```

Comment:
See other comment I think we likely should leave this here if possible.

Reply:
Makes sense. Done